### ISSUE TITLE
Addr is not reliable

### ISSUE THREAD

**Issue by Perlence** [ramitos/jsctags#43]
Addr holds the tag name surrounded by slashes, which is not enough to jump reliably. Consider this snippet:

```js
// foo
function foo () {}
```

When fed to `jsctags -f -` it yields:

```
foo     /home/sviatoslav/foo.js /foo/;" f       lineno:2        type:void function()
```

So this excmd `/foo/` will lead to first line with comment `// foo`, not function definition. The simplest solution is to use line number instead of search command like this:

```
foo     /home/sviatoslav/foo.js 2;"     f       type:void function()
```

**Comment by bigfish** [comment]
agreed this should be fixed.. vim is constantly jumping to the wrong spot because of this
